### DISCUSSION TITLE
[Suggestion] When EarTrumpet is already opened, clicking EarTrumpet icon from Start Menu should open Volume Mixer

### DISCUSSION THREAD

**Discussion by ghost** [File-New-Project/EarTrumpet#685] [Ideas]
If title was not clear enough, when EarTrumpet is running and user attempts to open a new instance of EarTrumpet, Volume Mixer should open up. That way we can pin EarTrumpet to Taskbar and have access to Volume Mixer whenever we want (for some reason I don't use keyboard shortcuts).

**Comment by 06000208** [comment]
I would greatly appreciate this. As it stands, the windows 7/10/11 volume mixers may be pinned to the taskbar, but EarTrumpet's cannot, afaik.

On windows 11 this would be especially useful, as taskbar tray icons may only be accessed from the the main monitor, but apps pinned to the taskbar appear on all monitors.
